src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/AttachCaseCallbackService.java

```diff
@@ -264,18 +281,29 @@ private void attachCaseByCcdId(
 
         StartEventResponse event = ccdApi.startAttachScannedDocs(theCase, idamToken, userId);
 
+        Map<String, Object> newCaseData = buildCaseData(newCaseDocuments, targetCaseDocuments);
+
         ccdApi.attachExceptionRecord(
             theCase,
             idamToken,
             userId,
-            buildCaseData(newCaseDocuments, targetCaseDocuments),
+            newCaseData,
             createEventSummary(theCase, exceptionRecordId, newCaseDocuments),
             event
         );
 
-        log.info("Attached exception record '{}' to case with CCD ID '{}'", exceptionRecordId, targetCaseCcdRef);
+        log.info(
+            "Successfully attach Exception Record with ID {} to case with id {}",
+            theCase.getId(),
+            exceptionRecord.getId()
+        );
+
+        paymentsProcessor.updatePayments(exceptionRecord, theCase.getId());
```

Comment:
Calling payments processor is very unexpected in a method that's supposed to update a case in CCD. It should be done after this method has completed.